proof_test.go

```diff
@@ -63,18 +63,18 @@ func verifyProof(t *testing.T, ap *AuthenticationPath, treeHash []byte, key stri
 	} else {
 		// proof of absence
 		// step 2. Check that where i and j differ is at bit l
```

Comment:
I think step 2 and step 3 should be switched. I don't think we should worry about checking the first ``l`` bits of the proof's index if ``vrf_verify`` returns false.